content/attacks/posts/2022-10-06-BSC-Token-Hub.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2022-10-06
+custodians: Binance Smart Chain
+categories: Smart contract
+title: "BSC Token Hub Hit By $586 Million Bridge Hack"
+---
+
+## Summary
+
+On October 6, 2022, BSC Token Hub, a bridge between BNB Beacon Chain (BEP2) and Binance Smart Chain (BEP20) was exploited. The native cross-chain bridge between BNB Beacon Chain (BEP2) and BNB Smart Chain (BEP20), also known as BNB Token Hub was exploited. The hacker used a low-level proof vulnerability and 2,000,000 $BNB were drained out of thin air. Consequently, the hacker began [bridging the funds to Fantom and Ethereum chains](https://www.coinbase.com/blog/bsc-token-hub-compromise-investigation-and-analysis-8). [The security experts in collaboration with validators were able to save the majority of the funds](https://www.bnbchain.org/en/blog/bnb-chain-ecosystem-update). The [hacker managed to bridge 127,000,000 $USD using AnySwap and Stargate bridges](https://rekt.news/bnb-bridge-rekt), with 53% of the stolen funds going to Ethereum, 33% to Fantom, and the rest to other chains. The remaining assets were left frozen in the attacker's address. 
+
+## Attacker
+
+Attackers address on BSC: 0x489a8756c18c0b8b24ec2a2b9ff3d4d447f79bec
+
+## Losses
+
+2 million BNB was minted, it equals $586 million. Only $127 million worth of assets were bridged, and the rest is left frozen. 
```

Comment:
can you link a source for this one?

Reply:
https://www.bitget.com/academy/BNB-Chain-586-Million-dollar-Hack-The-Horror-Before-Halloween
Sure, here it is, amounts may differ a little, cause of various token prices at the moment of calculations